device-types/Ubiquiti/UACC-AI-Port-RM.yaml

```diff
@@ -0,0 +1,55 @@
+---
+manufacturer: Ubiquiti
+model: AI Port Rack Mount
+slug: ubiquiti-uacc-ai-port-rm
+part_number: UACC-AI-Port-RM
+u_height: 1
+is_full_depth: false
+airflow: passive
+subdevice_role: parent
+weight: 650
+weight_unit: g
+module-bays:
```

Comment:
As the devices that go into this chassis are separately controlled entities these should be modeled as `device-bays` rather than `module-bays`